documentation/modules/deploying/con-securing-kafka-connect-api.adoc

```diff
@@ -0,0 +1,66 @@
+
+[role="_abstract"]
+To grant a user access to the Kafka Connect API, a Kafka cluster admin must provide them with the API URL. 
+This includes the hostname or IP address of the Kafka Connect cluster and the port number on which the API is running (by default, port 8083). 
+Additionally, the user must have authenticated access to the Kubernetes cluster and the Kafka Connect REST API endpoint in order to make a request.
+
+It is crucial to restrict access to the Kafka Connect API only to trusted users to prevent unauthorized actions and potential security issues. 
+The Kafka Connect API provides extensive capabilities for altering connector configurations, which makes it all the more important to take security precautions.
+
+If you are using the `KafkaConnector` custom resource to manage connectors, Strimzi's RBAC rules limit permission to make changes to Kubernetes cluster administrators by default.
+You can also xref:adding-users-the-strimzi-admin-role-str[designate non-cluster administrators to manage Strimzi resources].  
+With `KafkaConnector` resources enabled in your Kafka Connect configuration, changes made directly using the Kafka Connect REST API are reverted by the Cluster Operator.
+If you are not using the `KafkaConnector` resource, the default RBAC rules do not limit access to the Kafka Connect API directly.
+If you want to limit access to the Kafka Connect REST API using Kubernetes RBAC, you need to set up additional RBAC rules. 
+
+For improved security, we recommend configuring the following properties for the Kafka Connect API:
+
+`org.apache.kafka.disallowed.login.modules`:: (Kafka 3.4 or later) Set the `org.apache.kafka.disallowed.login.modules` Java system property to prevent the use of insecure login modules. 
+For example, specifying `org.apache.kafka.common.security.kerberos.KerberosLoginModule` prevents the use of the Kafka `KerberosLoginModule`.
++
+.Example configuration for disallowing login modules
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true" 
+spec:
+  # ...
+  jvmOptions:
+    javaSystemProperties:
+      - name: org.apache.kafka.disallowed.login.modules
+        value: org.apache.kafka.common.security.kerberos.KerberosLoginModule
+# ...      
+----
++
+System properties are passed to the JVM using the standard `-D` option.
++
+Only allow trusted login modules and follow the latest advice from Kafka for the version you are using.
+As a best practice, you should explicitly disallow insecure login modules in your Kafka Connect configuration by using the `org.apache.kafka.disallowed.login.modules` system property.
+
+`connector.client.config.override.policy`:: Set the `connector.client.config.override.policy` property to `None` (default) to prevent connector configurations from being overridden by unauthenticated requests. 
```

Comment:
Maybe this would work better?

```suggestion
`connector.client.config.override.policy`:: Set the `connector.client.config.override.policy` property to `None` (default) to prevent connector configurations to override the configuration of Kafka Connect and the consumers and producers used by it.
Before setting it to `None`, make sure that any connectors you have already deployed do not depend on the custom configuration.
```